src/components/LeftPanel/index.tsx

```diff
@@ -22,7 +24,7 @@ interface LeftPanelProps {
 
 const LeftPanel = ({ listItems }: LeftPanelProps) => {
   const version: string = `Version: ${env.REACT_APP_NRFESAMPLEAPP_VERSION}`;
-
+  
```

Comment:
I'm not sure if this is a lint issue or miss configuration, but for me this trailing spaces gives me a warning.

Reply:
It came from a small conflict that I resolved directly on git, it is fixed now! 💯